### PR TITLE
[UX] Open Github on clicking "Send feedback" under settings

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
* Uses custom tabs for opening urls
 * Opens the page to create new bugs / feature request on clicking the "send feedback" button

@gino-m  PTAL?

### CODE REVIEW

File: gnd/src/main/res/values/strings-untranslated.xml [hunk 1]
```diff
@@ -21,5 +21,6 @@
   <string name="app_name" translatable="false">Ground</string>
   <string name="text_separator" translatable="false">·</string>
   <string name="ground_website" translatable="false">https://groundplatform.org/</string>
+  <string name="ground_android_github" translatable="false">https://github.com/google/ground-android/issues/new/choose</string>
```
Comment (scolsen) [inline gnd/src/main/res/values/strings-untranslated.xml:24]: Maybe we should default to **bug report**? I suppose users could also want to request a feature, but they definitely won't report code health issues. I think reporting bugs is the most likely use case.

**Review by scolsen — APPROVED** [review]

**Comment by scolsen** [comment]
I re-triggered the continuous build since it seemed to potentially be a transient error (gradle daemon crashed).

File: gnd/src/main/res/values/strings-untranslated.xml (hunk 1 above)
Comment (shobhitagarwal1612) [reply gnd/src/main/res/values/strings-untranslated.xml:24]: @gino-m Please share your thoughts too. My rationale behind this was to promote users to suggest features in addition to reporting bugs.

**Review by gino-m — CHANGES_REQUESTED** [review]
Not so sure how useful this feature will be when working offline. Also, users need to have a GitHub account + permission to create issues in our GitHub repo. Is there another mechanism we can use to take feedback? In particular, we'd like to make it easy for users to report bugs, providing minimal friction.

**Comment by shobhitagarwal1612** [comment]
That's a great point! We could instead take the feedback directly from the app and send it to a common email whenever internet gets available.

**Comment by shobhitagarwal1612** [comment]
@gino-m @scolsen How about saving the feedback to a new firebase table similar to how responses are synced?

**Comment by shobhitagarwal1612** [comment]
Closing this for now. Please reopen if this is still relevant

**Pull request closed without merging**